README.md

```diff
@@ -28,6 +28,12 @@ Using npm:
 
     npm install --save react-stripe-elements
 
+Using UMD build (exports a global `ReactStripeElements` object):
+
+```html
+<script src="https://unpkg.com/react-stripe-elements@latest/dist/react-stripe-elements.min.js"></script>
+```
+
 Then, load Stripe.js in your application:
```

Comment:
Hmm, just the headers is still a bit clunky. What do you think about making the different installation methods styled as bullet points? Would look like this

<img width="920" alt="screen shot 2017-07-12 at 2 59 11 pm" src="https://user-images.githubusercontent.com/3409645/28141733-d20c56f4-6712-11e7-8797-28ff044163d3.png">